WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -599,6 +608,47 @@ - (void)visibilitySwitchAction:(id)sender
 {
     UISwitch *switcher = (UISwitch *)sender;
     Blog *blog = [self.resultsController objectAtIndexPath:[NSIndexPath indexPathForRow:switcher.tag inSection:0]];
+    if(!switcher.on && [self.tableView numberOfRowsInSection:0] > HideAllMinSites) {
+        if (self.hideCount == 0) {
+            self.firstHide = [NSDate date];
+        }
+        self.hideCount += 1;
+
+        if (self.hideCount > HideAllSitesThreshold && (self.firstHide.timeIntervalSinceNow * -1) < HideAllSitesInterval) {
```

Comment:
I just realized that the prompt only appears after reaching the threshold plus one.  Could we tweak this conditional be greater than or equal to the threshold?
`self.hideCount >= HideAllSitesThreshold`?